index.html

```diff
@@ -1,66 +1,147 @@
+          d = false;
+        tk.src = '//use.typekit.net/' + config.kitId + '.js';
+        tk.type = 'text/javascript';
+        tk.async = 'true';
+        tk.onload = tk.onreadystatechange = function () {
+          var a = this.readyState;
+          if (d || (a && a != 'complete' && a != 'loaded')) return;
+          d = true;
+          clearTimeout(t);
+          try {
+            Typekit.load(config);
+          } catch (b) {}
+        };
+        var s = document.getElementsByTagName('script')[0];
+        s.parentNode.insertBefore(tk, s);
+      })();
+    </script>
+  </head>
+  <body>
+    <div class="content--outer-wrap">
+      <div class="content--inner-wrap">
+        <h1 class="hide-text logo"><span>Leamington Geeks</span></h1>
+        <h2>
+          A free, friendly meet-up in Leamington Spa for<br /><span
+            id="js-people"
+            class="person"
+            >geeks</span
+          >
+        </h2>
+        <p class="sub-head">
+          Join us on
+          <span id="js-date-next">the last Monday of the month</span> from
+          7:30pm until late.<br />
+          <span id="js-days-to-go">&nbsp;</span>
+        </p>
+        <p><span id="js-date-next-next">&nbsp;</span></p>
+        <h2>Where?</h2>
+        <p>
+          Find us at
+          <a href="https://www.thewhitehorseleamingtonspa.co.uk"
+            >The White Horse</a
+          >
+          in Leamington Spa.<br />
+          We are usually in the outside seating area. It's a good idea to at
+          mention us on Twitter, if you are coming for the first time.
+        </p>
+        <p>
+          Calendar feed:
+          <a
+            href="https://www.google.com/calendar/ical/gqbti498rr2hijm6mhu0pteaus%40group.calendar.google.com/public/basic.ics"
+            >iCal</a
+          >
+        </p>
+        <p>
+          We post reminders about the next event on our Twitter:
+          <a href="https://twitter.com/leamgeeks"
+            >follow Leamington Geeks on twitter</a
+          >.
+        </p>
+
+        <h2>Getting Here</h2>
+        <p></p>
+        <h3>Bus</h3>
+        <p>
+          <a href="https://www.stagecoachbus.com/">Stagecoach</a> and
+          <a href="https://nxbus.co.uk/routes/coventry/C011/"
+            >National Express</a
+          >
+          operate services from Kenilworth, Warwick &amp; Coventry to
+          Leamington. The upper parade stop is the closest one to the White
+          Horse.
+        </p>
+        <h3>Train</h3>
+        <p>
+          <a href="https://www.thetrainline.com/stations/leamington-spa"
+            >Leamington Spa train station</a
+          >
+          is about 15 mins walk from the White Horse.
+        </p>
+        <h3>Car</h3>
+        <p>
+          The best places to park are on Clarendon Avenue outside the pub or
+          Beauchamp Road.
+        </p>
+        <p><a href="/code-of-conduct.html">Code of Conduct</a></p>
```

Comment:
Sorry about the formatting making this diff noisy but this is line is the only functional change.